main.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'app'
+
+ACTIONS = {
+  1 => :list_books,
+  2 => :list_people,
+  3 => :create_person,
+  4 => :create_book,
+  5 => :create_rental,
+  6 => :list_rentals_for_person,
+  7 => :exit_app
+}.freeze
+
+def display_menu
+  puts "\nPlease choose an option:"
+  ACTIONS.each { |key, value| puts "#{key} - #{value.to_s.tr('_', ' ')}" }
+  print 'Option: '
+  gets.chomp.to_i
+end
+
+def handle_option(app, option)
+  action = ACTIONS[option]
+  if action
+    app.send(action)
+  else
+    puts 'Invalid option. Please try again.'
+  end
+end
+
+def main
+  app = App.new
```

Comment:
- [ ] [OPTIONAL]
Let's endeavor to remove the marked portion from the console app so that the console looks more organized.

![image](https://github.com/Alibaba2023/oop-school/assets/12745474/4315b843-d1bb-450f-9e19-0e28765fea1a)